oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -58,12 +59,13 @@ def get_computer_system(uuid):
     """
 
     try:
-        resource = _get_oneview_resource(uuid)
+        resource_id, encl_id = zone.split_base_id_and_enclosure_id(uuid)
+        resource = _get_oneview_resource(resource_id)
         category = resource["category"]
 
         if category == 'server-profile-templates':
-            computer_system_resource = CapabilitiesObject(resource)
-        elif category == 'server-profiles':
+            computer_system_resource = CapabilitiesObject(uuid, resource)
```

Comment:
This will impact POST to computer system since the Id will be <template_uuid>-<enclosureid>. Please update blueprints/computer_system.py line 225: spt = g.oneview_client.server_profile_templates.get(body["Id"]) to retrieve the template.
Also api/computer_system.py method build_server_profile should save the id in the Description and not template uri, also where Description is read should be changed.

Reply:
As agreed, this will be fixed in another PR.